airflow/contrib/operators/slack_webhook_operator.py

```diff
@@ -37,6 +37,8 @@ class SlackWebhookOperator(SimpleHttpOperator):
     :type webhook_token: str
     :param message: The message you want to send on Slack
     :type message: str
+    :param attachments: The attachments you want to send on Slack
+    :type attachments: list
```

Comment:
in the similar spirit as above code-review, this doc string could also be a made a little verbose.

Reply:
Updated.